buildImages/build.gradle

```diff
@@ -19,12 +19,22 @@ if (!imageArch) {
 }
 rootProject.ext.imageArch = imageArch
 
+def commonJvmFlags = [
+        "-XX:MaxRAMPercentage=80.0",
+        "-XX:+ExitOnOutOfMemoryError",
+        "-XshowSettings:vm"
+]
+
 def jibProjects = [
         "TrafficCapture:trafficReplayer": [
                 baseImageName : "amazoncorretto",
                 baseImageTag  : "17-al2023-headless",
                 imageName     : "traffic_replayer",
                 imageTag      : "latest",
+                jvmFlags      : [
+                        // By default enable transparent proxy with older clients for maximum compatibility
+                        "-Dio.netty.handler.codec.http.defaultStrictLineParsing=false",
```

Comment:
Same question as before - how does jib wire up the java properties - are they env variables?  How would a user override this?